core/src/main/java/forklift/connectors/ForkliftConnectorI.java

```diff
@@ -33,32 +33,10 @@
      * @param source the source to read from
      * @return a consumer that reads from the given source
      * @throws ConnectorException if an error occurred interacting with the connector
-     * @throws RuntimeException if reading from the given source is not supported
+     * @throws RuntimeException   if reading from the given source is not supported
      */
     ForkliftConsumerI getConsumerForSource(SourceI source) throws ConnectorException;
 
-    /**
-     * Retrieves the {@link ForkliftConsumerI consumer} instance that reads from
-     * the queue with the given name.
-     *
-     * @param name the name of the topic to read from
-     * @return a consumer that reads from the given topic
-     * @throws ConnectorException if an error occurred interacting with the connector
-     * @throws RuntimeException if reading from a queue is not supported
-     */
-    ForkliftConsumerI getQueue(String name) throws ConnectorException;
```

Comment:
Might be good to stub these in to call getConsumerForSource to avoid backward compatibility issues.

Reply:
How much of a concern is this given that these methods are mostly internal?  My rationale for removing these is that more and more functionality is moving into the "Source" concept and if keep these methods around we could have diverging logic in places.